ods_ci/tests/Resources/Page/Operators/OpenShiftPipelines.resource

```diff
@@ -17,45 +17,18 @@ ${REDHAT_OPENSHIFT_PIPELINES_YAML}      ods_ci/tests/Resources/Files/redhat-open
-    ...     - OpenShift 4.11:                      OpenShift Pipelines 1.9
-    ...     - OpenShift 4.12, 4.13 (AWS and GCP):  OpenShift Pipelines 1.10 / 1.11 (depending on platform and flavor)
-    ...     - OpenShift >= 4.12 (OpenStack):       OpenShift Pipelines 1.11
-    ...     - OpenShift 4.14:                      OpenShift Pipelines 1.12
-    ...     Note: the complete operator compatiblity matrix is published at
-    ...      https://docs.openshift.com/container-platform/4.13/cicd/pipelines/op-release-notes.html
-    ${oc_version}=    Get OpenShift Version
-    ${is_self_managed}=    Is RHODS Self-Managed
-    ${platform}=    Fetch Cluster Platform Type
-    IF    "4.11" in "${oc_version}"
-        ${openshift_pipelines_version}=    Set Variable    pipelines-1.9
-    ELSE IF    "4.12" in "${oc_version}" or "4.13" in "${oc_version}"
-        IF    ${is_self_managed} == ${TRUE} and "${platform}" == "AWS"
-            ${openshift_pipelines_version}=    Set Variable    pipelines-1.10
-        ELSE IF    ${is_self_managed} == ${FALSE} and "${platform}" == "AWS"
-            ${openshift_pipelines_version}=    Set Variable    pipelines-1.11
-        ELSE IF    ${is_self_managed} == ${TRUE} and "${platform}" == "GCP"
-            ${openshift_pipelines_version}=    Set Variable    pipelines-1.11
-        ELSE IF    ${is_self_managed} == ${FALSE} and "${platform}" == "GCP"
-            ${openshift_pipelines_version}=    Set Variable    pipelines-1.10
-        ELSE
-            ${openshift_pipelines_version}=    Set Variable    pipelines-1.12
-        END
-    ELSE IF    "4.14" in "${oc_version}"
-        ${openshift_pipelines_version}=    Set Variable    pipelines-1.12
-    ELSE
-        ${openshift_pipelines_version}=    Set Variable    pipelines-1.12
-    END
-    Log    message=Installing OpenShift Pipelines operator: ${openshift_pipelines_version} (${platform} ${oc_version} Self-Managed: ${is_self_managed})
-    ...    console=True
-
+    [Documentation]    Install the latest Red Hat OpenShift Pipelines Operator by default. A different version
+    ...    can be installed by setting OPENSHIFT_PIPELINES_VERSION in test-variables.yml .
+    ...    Example: if the value of OPENSHIFT_PIPELINES_VERSION is pipelines-1.12 will install the latest 1.12 version
+    ...    that currently is 1.12.2
     ${return_code}=    Run And Return Rc
-    ...    sed -i "s,channel: .*,channel: ${openshift_pipelines_version},g" ${EXECDIR}/${REDHAT_OPENSHIFT_PIPELINES_YAML}
+    ...    sed -i "s,channel: .*,channel: ${OPENSHIFT_PIPELINES_VERSION},g" ${EXECDIR}/${REDHAT_OPENSHIFT_PIPELINES_YAML}
     Should Be Equal As Integers    ${return_code}    0
     Oc Apply    kind=Subscription    src=${REDHAT_OPENSHIFT_PIPELINES_YAML}
     ${pod_count}=    Wait Until OpenShift Pipelines Operator Is Deployed
     Should Be True    ${pod_count} == 1    msg=Error installing OpenShift Pipelines operator
+    ${res}  ${output}=    Run And Return Rc And Output
```

Comment:
don't you want to check the output?

Reply:
I am using the line below `Log     A OpenShift Pipelines Operator of version ${output} is installed    console=True`

Reply:
@apodhrad the idea was not check the version, right ?

Reply:
Hi @diegolovison my original suggestion was to skip any installation if it is already installed, something like
```
is any pipelines-operator installed?
then
    use it and continue
else
    install the latest one
```
in this case you don't need to check any version.

But it seems you try to allow users to specify the channel which is ok but it add other complexity
* you need to check the version as Berto has pointed out
* you need to add another variable into test-variables.yaml

Reply:
We don't need the if. If installed, even executing the code it won't install or replace the version

Reply:
awesome!

Reply:
@apodhrad @bdattoma in this case, should I check or not the version? what do you prefer?

Reply:
regarding the version checking.. .when we rename the variable `OPENSHIFT_PIPELINES_VERSION` to `OPENSHIFT_PIPELINES_CHANNEL` then we do not know the version (in theory we could deduced it) but IMHO it is not necessary.

Reply:
@bdattoma ?

Reply:
okay. But keep at least the Log you did, so we could use it for debug